poc_iot_verifier/src/meta.rs

```diff
@@ -58,12 +58,9 @@ impl Meta {
         .await?
         .and_then(|v| {
             v.parse::<u64>()
-                .map_or_else(|_| None, |secs| Some((secs + 10).to_timestamp()))
-            //TODO: remove the hardcoded + 10 above
-            //      fix resolution of datetime function, dropping millisecs resulting in files being reprocessed by loaded
-        })
-        .transpose()?;
-        Ok(height)
+                .map_or_else(|_| None, |ts| Some(Utc.timestamp_millis(ts as i64)))
```

Comment:
Use the `TimestampDecode` trait for u64

Reply:
fixed here: 22bbdbe0ff321c5eeb1c4661758329296a028188